### DISCUSSION TITLE
Possibilidade de separar ambientes entrre develop e main

### DISCUSSION THREAD

**Discussion by Flamilani** [techexpertspro/portal-ascepa#38] [Ideas]
Temos que pensar possibilidade de separar ambientes entrre develop e main para facilitar organização de branchs e revisão de código

**Comment by andrewarosario** [comment]
Para agilizar os processos e dar mais autonomia para vocês (para que nós, mentores, não sejamos um gargalo) nós estamos utilizando atualmente o [Trunk Based Development](https://www.objective.com.br/insights/trunk-based-development/).
Com essa ideia não teríamos uma branch `develop`. Basicamente vocês criam branches com base na `main` e sobem seus Pull Requests. A ideia é que trabalhem com desenvolvimento de pequenos lotes, sempre com Review e Integração Contínua.

Mas gostaria de ouvir vocês. Acham que um fluxo mais baseado no Gitflow traria mais benefícios? Bora discutir 💬

**Reply by Flamilani** [reply]
Boa dica para entender a estratégia de branch. Vou dar estudada sobre Trunk Based Development.
